Core/include/Acts/Visualization/PlyVisualization.hpp

```diff
@@ -62,7 +68,19 @@ class PlyHelper : public IVisualization {
   }
 
   /// @copydoc Acts::IVisualization::write()
-  void write(std::ostream& os) const override {
+  void write(const std::string& path) const final {
+    std::ofstream os;
+    std::string objectpath = path;
+    if (not IVisualization::hasExtension(path)) {
+      objectpath += std::string(".ply");
+    }
```

Comment:
Could this not use `IVisualization::replaceExtension` if you have that?

Reply:
No, this just adds one in case there is none.

Reply:
The case here covers in case there's none.